### PR TITLE
TC02

### PR DESCRIPTION
**Pull request by trangnhomanhcoi2601**

### CODE REVIEW

File: src/main/java/helpers/DriverHelper.java [hunk 1]
```diff
@@ -5,13 +5,16 @@
...
+import java.util.concurrent.TimeUnit;
+
 public class DriverHelper {
 
     private static WebDriver driver;
 
     public static void initDriver() {
         WebDriverManager.chromedriver().setup();
         driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Constants.TIME_WAIT, TimeUnit.SECONDS);
```
Comment (anhlvt) [inline src/main/java/helpers/DriverHelper.java]: We should not use implicit wait in project, please research more about this wait

File: src/test/java/tests/LoginTests.java [hunk 2]
```diff
@@ -21,6 +21,16 @@ public void tc01_LoginWithValidData() {
...
+    }
+
+    @Test(description = "User can't login with blank 'Username' textbox")
+    public void tc02_LoginWithBlankUsernameTextbox() {
+        LogHelper.info("Don't enter user name. enter valid information into 'Password' textbox");
+        loginPage.login("", Constants.PASSWORD);
+
+        LogHelper.info("Check error messages");
+        Assert.assertEquals(loginPage.getErrorEmailMessage(), "You must specify a username.", "Error email message doesn't display correctly");
+        Assert.assertEquals(loginPage.getErrorLoginMessage(), "There was a problem with your login and/or errors exist in your form.", "Error message doesn't display correctly");
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTests.java]: Error login message doesn't display...

File: src/test/java/tests/LoginTests.java [hunk 3]
```diff
@@ -21,6 +21,16 @@ public void tc01_LoginWithValidData() {
         loginPage.login(Constants.USER, Constants.PASSWORD);
 
         LogHelper.info("Check welcome user message");
-        Assert.assertEquals(loginPage.getWelcomeUserMessage(), "Welcome " + Constants.USER, "Welcome user message doesn't display correctly.");
+        Assert.assertEquals(loginPage.getWelcomeUserMessage(), "Welcome " + Constants.USER, "Welcome user message doesn't display correctly");
+    }
+
+    @Test(description = "User can't login with blank 'Username' textbox")
+    public void tc02_LoginWithBlankUsernameTextbox() {
+        LogHelper.info("Don't enter user name. enter valid information into 'Password' textbox");
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTests.java]: It should be an uppercase letter after dot

Comment (trangnhomanhcoi2601) [reply src/test/java/tests/LoginTests.java]: I've already fixed it.

File: src/test/java/tests/LoginTests.java (hunk 2 above)
Comment (trangnhomanhcoi2601) [reply src/test/java/tests/LoginTests.java]: I've already fixed it.

File: src/main/java/helpers/DriverHelper.java (hunk 1 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/helpers/DriverHelper.java]: I've already fixed it.

**Review by anhlvt — APPROVED** [review]

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**